sql/synth_recipes.sql

```diff
@@ -2553,7 +2553,7 @@ INSERT INTO `synth_recipes` VALUES (34527,0,0,0,0,0,94,0,0,0,0,4099,4241,9130,10
 INSERT INTO `synth_recipes` VALUES (34528,0,0,0,0,0,94,0,0,0,0,4099,4241,816,830,830,0,0,0,0,0,13587,13627,13627,13627,1,1,1,1,'Rainbow Cape');
 -- INSERT INTO `synth_recipes` VALUES (34529,0,2011,0,0,0,95,0,0,0,0,4098,4240,2145,2145,2145,2145,2145,2145,4016,4016,4016,4016,4016,4016,3948,3948,3948,3948,18,24,30,36,'Tulfaire Fletchings');
 INSERT INTO `synth_recipes` VALUES (34530,0,0,0,0,0,95,0,0,0,0,4099,4241,1713,1713,1713,0,0,0,0,0,1714,1714,1714,1714,1,1,1,1,'Cashmere Cloth');
--- INSERT INTO `synth_recipes` VALUES (34531,0,0,0,0,50,95,50,0,0,0,4099,4241,828,828,2152,2340,2340,2473,2476,0,2455,2456,2456,2456,1,1,1,1,'Cursed Trews');
+INSERT INTO `synth_recipes` VALUES (34531,0,0,0,0,45,95,31,0,0,0,4099,4241,828,828,2152,2340,2340,2473,2476,0,2455,2456,2456,2456,1,1,1,1,'Cursed Trews'); -- wiki, BG, jpwiki, AH checked 24/Nov/2021
```

Comment:
Same here. BG wiki has 50 in both. If you have the JPwiki values, RE the link, pls.

Reply:
https://wiki-ffo-jp.translate.goog/html/12175.html?_x_tr_sl=auto&_x_tr_tl=en&_x_tr_hl=en-US&_x_tr_pto=nui&_x_tr_sch=http

Same reasons like above.